docs/pages/versions/unversioned/sdk/stripe.md

```diff
@@ -66,6 +66,11 @@ For usage information and detailed documentation, please refer to:
 If you're relying on redirects, you'll need to pass in a `urlScheme` to `initStripe`. To make sure you always use the proper `urlScheme`, pass in:
 
 ```js
+import * as Linking from 'expo-linking';
+import Constants, { ExecutionEnvironment } from 'expo-constants';
```

Comment:
```suggestion
import Constants from 'expo-constants';
```
`master` no longer uses `ExecutionEnvironment `

Reply:
@cruzach Thanks for noticing. Things changing fast 😅